gax-java/gax/src/main/java/com/google/api/gax/retrying/RetrySettings.java

```diff
@@ -166,36 +197,53 @@ public abstract static class Builder {
 
     /**
      * TotalTimeout has ultimate control over how long the logic should keep trying the remote call
-     * until it gives up completely. The higher the total timeout, the more retries can be
+     * until it gives up completely. The higher the total timeout, the more retries and polls can be
      * attempted. The default value is {@code Duration.ZERO}.
+     *
+     * <p>If there are no configurations, Retries have the default timeout value set above and LROs
+     * have a default total timeout value of {@code Duration.ofMillis(300000)} (5 minutes).
      */
     public abstract Builder setTotalTimeout(Duration totalTimeout);
 
     /**
-     * InitialRetryDelay controls the delay before the first retry. Subsequent retries will use this
-     * value adjusted according to the RetryDelayMultiplier. The default value is {@code
-     * Duration.ZERO}.
+     * InitialRetryDelay controls the delay before the first retry/ poll. Subsequent retries and
+     * polls will use this value adjusted according to the RetryDelayMultiplier. The default value
+     * is {@code Duration.ZERO}.
+     *
+     * <p>If there are no configurations, Retries have the default initial retry delay value set
+     * above and LROs have a default initial poll delay value of {@code Duration.ofMillis(5000)} (5
+     * seconds).
      */
     public abstract Builder setInitialRetryDelay(Duration initialDelay);
 
     /**
-     * RetryDelayMultiplier controls the change in retry delay. The retry delay of the previous call
-     * is multiplied by the RetryDelayMultiplier to calculate the retry delay for the next call. The
-     * default value is {@code 1.0}.
+     * RetryDelayMultiplier controls the change in delay before the next retry or poll. The retry
+     * delay of the previous call is multiplied by the RetryDelayMultiplier to calculate the retry
+     * delay for the next call. The default value is {@code 1.0}.
+     *
+     * <p>If there are no configurations, Retries have the default retry delay multiplier value set
+     * above and LROs have a default retry delay multiplier of {@code 1.5}.
      */
     public abstract Builder setRetryDelayMultiplier(double multiplier);
 
     /**
      * MaxRetryDelay puts a limit on the value of the retry delay, so that the RetryDelayMultiplier
      * can't increase the retry delay higher than this amount. The default value is {@code
      * Duration.ZERO}.
+     *
+     * <p>If there are no configurations, Retries have the default max retry delay value set above
+     * and LROs have a default max poll retry delay value of {@code Duration.ofMillis(45000)} (45
+     * seconds).
      */
     public abstract Builder setMaxRetryDelay(Duration maxDelay);
 
     /**
      * MaxAttempts defines the maximum number of attempts to perform. The default value is {@code
      * 0}. If this value is greater than 0, and the number of attempts reaches this limit, the logic
```

Comment:
I remember that both 0 and 1 mean there is no retry from a YAQS. It's kind of already implied as `maxAttempts` should include the first attempt, can you please make it clear in the comment here to reduce similar confusion in the future?

Reply:
Updated. Let me know if this makes more sense.

Reply:
`The first attempt will be considered attempt #0.` I think this is true if `maxAttempts` is set to 0, but `The first attempt will be considered attempt #1` if `maxAttempts` is set to 1?

Reply:
The first attempt will be set as Attempt #0 (https://github.com/googleapis/sdk-platform-java/blob/80f23ec98922bf8f29f3f6960fb20fbeb6231706/gax-java/gax/src/main/java/com/google/api/gax/retrying/ExponentialRetryAlgorithm.java#L104). Every retryAttempt will create the `nextAttempt` class and update the attemptCount: https://github.com/googleapis/sdk-platform-java/blob/80f23ec98922bf8f29f3f6960fb20fbeb6231706/gax-java/gax/src/main/java/com/google/api/gax/retrying/ExponentialRetryAlgorithm.java#L164 and the shouldRetry function inside the algorithm will check if it's allowed. In this case, the first _retry_ attempt will be attempt #1 and that will pass this check: https://github.com/googleapis/sdk-platform-java/blob/80f23ec98922bf8f29f3f6960fb20fbeb6231706/gax-java/gax/src/main/java/com/google/api/gax/retrying/ExponentialRetryAlgorithm.java#L224-L226.

I think it would make sense if I specify that maxAttempts refers to the number of retry attempts instead.